sources/mk/macedonia-municipalities.hjson

```diff
@@ -0,0 +1,136 @@
+    attribution: [
+      {
+        label: {
+          en: © OpenStreetMap contributors
+        }
+        url: {
+          en: https://www.openstreetmap.org/copyright
+        }
+      }
+      {
+        label: {
+          en: Elastic Maps Service
+        }
+        url: {
+          en: https://www.elastic.co/elastic-maps-service
+        }
+      }
+    ]
+    fieldMapping: [
+        {
+            type: id
+            name: iso_3166_2
+            desc: ISO-3166-2 identifier
+        }
+        {
+            type: property
+            name: label_en
+            desc: Subdivision name (English)
+        }
+        {
+            type: property
+            name: label_mk
+            desc: Subdivision name (Macedonian)
+        }
+        {
+            type: property
+            name: label_sq
+            desc: Subdivision name (Albanian)
+        }
+    ]
+    query: {
+        sparql:
+            '''
+            #defaultView:MapRegions
+            SELECT
+            (if(bound(?id2),?id2,?id) as ?id)
+            ?iso_3166_2 ?label_en ?label_mk ?label_sq ?wkidataOSM
+            WHERE {
+              SERVICE <https://query.wikidata.org/sparql> {
+              # Using nested query to ensure there is only one ?id2 value
+              {SELECT
+              ?id
+              (SAMPLE(?id2) as ?id2)
+              (SAMPLE(?iso_3166_2) as ?iso_3166_2)
+              (SAMPLE(?label_en) as ?label_en)
+              (SAMPLE(?label_mk) as ?label_mk)
+              (SAMPLE(?label_sq) as ?label_sq)
+              WHERE {
+                # List of regions, whose sub-regions we want.
+                VALUES ?entity { wd:Q221 }
+
+                # P150 = "contains administrative territorial entity"
+                ?entity wdt:P150* ?id .
+                ?id wdt:P300 ?iso_3166_2
+
+                    OPTIONAL { ?id rdfs:label ?label_en . FILTER(LANG(?label_en) = "en") }
+                    OPTIONAL { ?id rdfs:label ?label_mk . FILTER(LANG(?label_mk) = "mk") }
+                    OPTIONAL { ?id rdfs:label ?label_sq . FILTER(LANG(?label_sq) = "sq") }
+              }
+              # remove possible ID duplicates
+              GROUP BY ?id
+              ORDER BY ?id}
+            }
+            }
+
+            '''
+    }
+    name: macedonia_municipalities
+    legacyIds: [
+      Macedonia Municipalities
+    ]
+    humanReadableName: {
+      ar: بلدية في جمهورية مقدونيا
+      bar: Gmoana in Mazedonien
+      ca: Municipi de Macedònia del Nord
+      cs: Seznam opštin v Severní Makedonii
+      de: Opština in Nordmazedonien
+      el: Δήμοι της Βόρειας Μακεδονίας
+      en: North Macedonia Municipality
+      es: Municipio de Macedonia del Norte
+      fr: opština de la République de Macédoine
+      he: מחוזות מקדוניה
+      hr: Općine u Makedoniji
+      hy: Մակեդոնիայի Հանրապետության վարչական բաժանում
+      id: Munisipalitas di Makedonia Utara
+      ie: Liste de municipalitates de Macedonia
```

Comment:
Sometimes these labels come from tables in Wikipedia (e.g. List of...). Not a big deal since Kibana is not (and probably never will be) localized to [Interlingue](https://en.wikipedia.org/wiki/Interlingue). I have already fixed this in Wikidata.

```suggestion
      ie: Municipalitates de Macedonia
```